app-infrastructure/dictionary-instance.tf

```diff
@@ -1,10 +1,10 @@
-data "template_file" "dictionary-user_data" {
+data "template_file" "dictionary-user-data-template" {
```

Comment:
In our next infrastructure iteration we should standardize on `_` for objects in terra as noted in the google best practices document you introduced to me. https://github.com/hms-dbmi/pic-sure-bdc-infrastructure/pull/99/files